screens/FavPokemon.tsx

```diff
@@ -0,0 +1,52 @@
+import React, { useEffect, useState } from 'react';
+import { StyleSheet, Text, View, Image, Button } from 'react-native';
+import getFavPokemon from '../api/pokemon';
+import styles from '../style/base';
+import PropRow from '../components/PropRow';
+
+const FavPokemon = ({ navigation }) => {
+  const [favPokemon, setFavPokemon] = useState(null);
+  const [loading, setLoading] = useState(true);
+
+  useEffect(() => {
+    async function effect() {
```

Comment:
ah yes, there's a problem with useEffect and async/await, I forgot about it 😄 but it's nice that you figured out how to implement it 👍